xblock/reference/plugins.py

```diff
@@ -171,3 +130,42 @@ def __set__(self, xblock, value):
         We don't support this until we figure out what this means
```

Comment:
I guess that it just seems weird to me to move a field to "supported" when so many of the interactions with the field will be using dependencies that are implemented in an unsupported prototype. I'm not sure what the guarantees are that the "supported" aspect of the system is trying to provide, other than that an unsupported system also exists.